saleor/product/models/base.py

```diff
@@ -73,6 +73,7 @@ class ProductClass(models.Model):
         'ProductAttribute', related_name='products_class', blank=True)
     variant_attributes = models.ManyToManyField(
         'ProductAttribute', related_name='product_variants_class', blank=True)
+    is_shipping_required = models.BooleanField(default=False)
```

Comment:
We should include a label here and probably for other fields as well. Please note that `name` incorrectly claims to be a `Product field`.